src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,31 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTab, activeTabId, onTabSelected }) => {
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              className={tab.id === activeTabId ? 'is-active' : null}
+              data-cy="Tab"
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={
+                  tab.id === activeTabId ? null : () => onTabSelected(tab.id)
+                }
+              >
+                {tab.title}
+              </a>
+            </li>
+          ))}
+        </ul>
+      </div>
+
+      <div className="block" data-cy="TabContent">
+        {tabs.find(tab => tab.id === selectedTab)?.content}
```

Comment:
Instead of using this find here, do it outside the return, and than you can validate it to pass the tests.
Something like:
```
const displayTab = tabs.find(tab => tab.id === selectedTab) ?? tabs[0];

...
return (
    ...
    <div className="block" ... >
        {displayTab.content}
    </div>
    ...
```
This code will try finding the sab with the given id, and if not found, will return tabs[0] instead.